docker-compose.yml

```
@@ -101,9 +101,65 @@ services:
     networks:
       - mesachain-network
 
+  prometheus:
+     image: prom/prometheus:v2.52.0   # pin to a specific, tested version
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 5

(indentation)